### DISCUSSION TITLE
How to remove a module from waybar?

### DISCUSSION THREAD

**Discussion by sanjyay** [prasanthrangan/hyprdots#1460] [Q&A] [answered]
![image](https://github.com/prasanthrangan/hyprdots/assets/102979855/e1e7b4d9-3b3e-44d5-a6ea-37f972593f76)
I want to remove (switch wallpaper,switch theme,switch dock and cliphust )from waybar . I tried editing config.ctl in ~/.config/waybar 
from :
1|25|top|( cpu memory custom/cpuinfo ) ( idle_inhibitor clock ) ( hyprland/workspaces )|( hyprland/window )|( backlight network bluetooth pulseaudio pulseaudio#microphone custom/updates custom/keybindhint ) ( tray battery ) ( custom/wallchange custom/theme custom/wbar custom/cliphist custom/power )
to:
#1|25|top|( cpu memory custom/cpuinfo ) ( idle_inhibitor clock ) ( hyprland/workspaces )|( hyprland/window )|( backlight network bluetooth pulseaudio pulseaudio#microphone custom/updates custom/keybindhint ) ( tray battery ) (  custom/power )
But i could see no changes .

**Comment by kRHYME7** [comment]
toggle your waybar/change a layout.  If you want it to auto reload  run this 

` Hyde waybar control ` 
edit the config then if you save it will reload for you

**Reply by sanjyay** [reply]
Even after I toggle the layout  there is no change.If I try to change the waybar using Super+Alt+UP/DOWN still there is no change . After trying to change the waybar layout.The config file results this 
![image](https://github.com/prasanthrangan/hyprdots/assets/102979855/fa887dbd-75b6-475d-b639-cac98c22060a)

**Reply by kRHYME7** [reply, marked as answer]
Wait do you know that the active layout had ` 1| ` as the 1st field?  

Make your edits on the current layout with 1

**Reply by sanjyay** [reply]
My bad . I was copy pasting the same line below the active layout and was editing it . I have got it now. 
i have another question,if i have to add something to waybar like an option to toggle blue light filter then 
i will have to add a sh file to modules folder 
Make changes config.jsonc and config.ctl right?

**Reply by kRHYME7** [reply]
do not touch cofig.json and style.css because they are autogenerated so it's pointless 


add it to your config.ctl instead 

be sure that module is working and inside ./modules 

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/44ca941a-ff97-4229-a956-22d6a4b8599d)


just add ` custom/yourmodule ` in the config.ctl
